benchmarks/kubectl-mtb/README.md

```diff
@@ -0,0 +1,14 @@
+# Multi-Tenancy Benchmarks
+
+This is a kubectl plugin of the multi-tenancy benchmarks, to validate the the multi-tenancy in the K8s cluster.
+
+## Building the plugin
+
+Makefile is used to automate the things at the local level.
+**Prerequisites** : Make sure you have working GO environment in your system.
+
+### Make commands
+
+- **make kubectl** : To build the project and copy the binary file to the PATH.
```

Comment:
would suggest we call this "make mtb" or "make kubectl-mtb" to avoid confusion with kubectl.